static/styles/compose.css

```diff
@@ -276,51 +276,96 @@
     display: none;
 }
 
-.compose_resolved_topic,
-.compose_invite_user,
-.compose_private_stream_alert,
-.compose-all-everyone,
-.compose_not_subscribed {
-    padding: 4px 0;
+.compose_banner {
+    margin-bottom: 20px;
+    border-radius: 4px;
+    padding: 3px 5px 3px 15px;
+    border: 1px solid;
     display: flex;
+    align-items: center;
     justify-content: space-between;
-    position: relative;
-}
 
-#compose_not_subscribed_close {
-    top: 0;
-    right: 0;
-    position: absolute;
-}
+    p {
+        margin: 0; /* override bootstrap */
+        margin-right: 20px;
+        flex-grow: 1;
+    }
 
-.compose_resolved_topic_user_controls .compose_resolved_topic_close,
-.compose_invite_user_controls .compose_invite_close,
-.compose_private_stream_alert_controls .compose_private_stream_alert_close {
-    display: inline-block;
-    position: absolute;
-    margin-top: -2px;
-    margin-right: -2px;
-    width: 10px;
-}
+    .action_button {
```

Comment:
We do not want to be moving in this direction of unqualified CSS class names. They hinder modularity, even when “scoped” using nesting, because the CSS notion of scope doesn’t match with the conceptual notion of scope in components. See

https://chat.zulip.org/#narrow/stream/6-frontend/topic/CSS.20nesting/near/1283355